lib/jsonb_accessor/macro.rb

```diff
@@ -44,19 +44,21 @@ def jsonb_accessor(jsonb_attribute, field_types)
           jsonb_values = public_send(jsonb_attribute) || {}
           jsonb_values.each do |store_key, value|
             name = names_and_store_keys.key(store_key)
-            write_attribute(name, value)
+            write_attribute(name, value) if name.present?
```

Comment:
Well `name` is the key of the attribute so the kind of value shouldn't matter right?